libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/JourneyStringTransformer.kt

```diff
@@ -61,6 +77,19 @@ private const val V1API_JSON_JOLT_SPEC = """[{
         }
       }
     },
+    "poi": {
+      "*": { "\\@attributes": "pois" }
+    },
+   
     "error": "Error"
   }
-}]"""
+},
+// ... so we need to convert the strings back to arrays!
```

Comment:
wow this looks like it was tedious 😆

Reply:
- and painful... it took ages to work the whole thing out!